Lib/test/test_generators.py

```diff
@@ -268,6 +268,91 @@ def loop():
         #This should not raise
         loop()
 
+
+class ModifyTest(unittest.TestCase):
```

Comment:
```suggestion
class ModifyFrameLocalsTest(unittest.TestCase):
```
Name feels a bit too general

Reply:
This name is not correct, since we modify generators with two different methods for now: via frame locals and by creation new generator based on the same code object.

Reply:
Maybe "ModifyBaseIteratorTest"?
Or "ModifyUnderlyingIterableTest"?